AfsprakenRegels.md

```diff
@@ -461,31 +461,53 @@ aanduiding gedefinieerd worden.
+gelden. 
+
+Het is dan niet mogelijk om het te modelleren als relatieklasse. Wel gewenst, 
+maar het kan niet als UML-associationClass. 
+Als deze uitzondering het geval is, dan worden de relatieklasses gemodelleerd als «Objecttype», 
+met één inkomende relatie en één uitgaande relatie. 
+De oorspronkelijke kardinaliteit van de beoogde relatieklasse wordt hierbij behouden. 
+
+De relatie is geobjectiveerd. Elke betrokken relatie wordt in dit geval behandeld als een object met gegevens. 
 
 <pre class='example'>
 Een Perceel kan vanwege een Perceel splitsing overgaan in twee of meerdere
-andere Percelen. De ‘overgegaan in’ relatie wordt bijgehouden in een
-relatieklasse. Gegevens over de splitsing zijn voor al deze relaties gelijk.
+andere Percelen. De ‘overgegaan in’ relatie wordt bijgehouden als een
+relatieklasse met eigen (splitsing) gegevens. Bij een splitsing ontstaan * nieuwe percelen 
+en er is daarom sprake van meerdere relaties. De gegevens over de splitsing zijn echter altijd 
+voor al deze relaties gelijk. Om deze reden wordt de relatieklasse 'overgegaan in' gemodelleerd als een objecttype.  
 </pre>
 
-Het metamodel ondersteunt (nog) geen relatieklassen tussen drie of meer
-objecttypen. Dit kan in uw eigen extensie toegevoegd worden.
-
 <pre class='ex-generic'>
 Een CONTRACT kan bijvoorbeeld ook een afspraak zijn tussen twee
 óf méér SUBJECTen, waarbij de gegevens van de relatie voor alle betrokken
 objecten hetzelfde zijn. CONTRACT wordt dan gemodelleerd als objecttype, waarbij
-beschreven wordt wat er moet gebeuren wanneer één van de SUBJECTen niet meer
-bestaat.
+beschreven wordt wat er moet gebeuren wanneer één van de SUBJECTen niet meer bestaat.
 </pre>
 
+Opmerkingen: 
+- de gegevens over de relatie bestaan alleen zolang de relatie tussen beide objecten bestaat en zolang elk van beide objecten zelf (nog) bestaan. Dit is echter niet meer te zien aan de metaklasse, omdat dit «Objecttype» is geworden en niet meer «Relatieklasse» is. Geef daarom in de toelichting van het objecttype aan dat elk object van dit objecttype bestaansafhankelijk is van de beide objecttypen die verbonden zijn door deze geobjectiveerde relatie.  
+- normaal gesproken bij relaties tussen objecten wordt de relatie op instantie niveau geïdentificeerd door de identificaties van beide betrokken objecten. Er zijn geen aparte regels voor de opbouw van de identificatie van een relatieklasse (en ook niet voor geobjectiveerde relatieklassen die als objecttypen zijn gemodelleerd). Anders gezegd, een regel zoals dat de identificatie van een relatieklasse (of geobjectiveerde relatieklassen die als objecttypen zijn gemodelleerd) is opgebouwd uit de samenstelling van de identificatie van bron en doel objecten is niet aanwezig in de MIM specificatie. Evenzo bestaat er in MIM geen regel of best practise dat er een nieuwe betekenisloze identificatie toegekend moet worden. Het is aan de modelleur om hierin eigen afwegingen te maken. 
```

Comment:
> normaal gesproken bij relaties tussen objecten wordt de relatie op instantie niveau geïdentificeerd door de identificaties van beide betrokken objecten.

En de kenmerkreferentie! Relatie(gegevens) wordt net als alle andere gegevens altijd geidentificeerd door de combinatie van het onderwerp, de kenmerkreferentie en de waarde,

> Er zijn geen aparte regels voor de opbouw van de identificatie van een relatieklasse (en ook niet voor geobjectiveerde relatieklassen die als objecttypen zijn gemodelleerd). Anders gezegd, een regel zoals dat de identificatie van een relatieklasse (of geobjectiveerde relatieklassen die als objecttypen zijn gemodelleerd) is opgebouwd uit de samenstelling van de identificatie van bron en doel objecten is niet aanwezig in de MIM specificatie. Evenzo bestaat er in MIM geen regel of best practise dat er een nieuwe betekenisloze identificatie toegekend moet worden. Het is aan de modelleur om hierin eigen afwegingen te maken.

Volgens mij is de identificatie van een geïnstantieerde relatieklasse precies hetzelfde als die van een relatie. Een relatieklasse is namelijk semantisch gelijk aan een relatiesoort.

Maar het zou goed zijn om te beschrijven wat we vinden dat die regel is.